packages/next-codemod/bin/upgrade.ts

```diff
@@ -412,23 +412,34 @@ function isUsingAppDir(projectPath: string): boolean {
  */
 async function suggestTurbopack(packageJson: any): Promise<void> {
   const devScript: string = packageJson.scripts['dev']
-  if (devScript?.includes('--turbopack')) return
 
-  const responseTurbopack = await prompts(
-    {
-      type: 'confirm',
-      name: 'enable',
-      message: 'Enable Turbopack for next dev?',
-      initial: true,
-    },
-    { onCancel }
-  )
-
-  if (!responseTurbopack.enable) {
+  if (!devScript) {
+    console.log(
+      `${pc.red('⨯')} Could not find a "dev" script in your package.json.`
+    )
     return
   }
 
   if (devScript.includes('next dev')) {
+    // covers "--turbopack" as well
+    if (devScript.includes('--turbo')) {
```

Comment:
Good call to add a comment here. We might need to be more fine-grained here in the future so important to ensure we always handle `--turbo` and `--turbopack`